test/Microsoft.AspNetCore.Razor.Evolution.Test/IntegrationTests/RazorIRNodeWriter.cs

```diff
@@ -147,7 +147,10 @@ protected void WriteName(RazorIRNode node)
 
         protected void WriteLocation(RazorIRNode node)
         {
-            _writer.Write(node.SourceLocation.ToString());
+            if (node.DocumentLocation != null)
```

Comment:
So I guess what I mean to say is that we should do something simple that works 👍 until we have more time to look at it.